tutorials/3d/particles/attractors.rst

```diff
@@ -13,6 +13,8 @@ and :ref:`class_GPUParticlesAttractorVectorField3D`. You can instantiate them at
 change their properties from gameplay code; you can even animate and combine them for complex
 attraction effects.
 
+.. UPDATE: Not implemented.
```

Comment:
```suggestion
.. UPDATE: Not implemented. Remove this comment when particle attractors are implemented for 2D particle systems.
```
Comments should maybe be more explicit about to do when the information changes.